### PR TITLE
Aave history [sc-10793]

### PR DESCRIPTION
**Pull request by paszkowskiDamian**
# [Aave history](https://app.shortcut.com/oazo-apps/story/10793/frontend-add-subgraph-query-to-get-position-events)
  
## Changes 👷‍♀️
- fetch aave data from subgraph
- new env variable added `AAVE_SUBGRAPH_URL`
  
## How to test 🧪

example vault ids to check: `322`,`407`

image

image
image
image
image
image
image
image

TODO:

I still need to figure out that `maxCoverage` means in automation.

### CODE REVIEW

File: features/ajna/history/types.ts [hunk 1]
```diff
@@ -41,5 +49,12 @@ export type AjnaBorrowerEvent = {
 
 // TODO to be removed when implementing aave history, dummy aave history interface
 export interface AaveHistoryEvent extends PositionHistoryEvent {
-  aaveStuff: BigNumber
+  quoteTokensAfter?: undefined
```
Comment (jakubswierczek) [inline features/ajna/history/types.ts:52]: please add types

File: features/ajna/history/types.ts [hunk 2]
```diff
@@ -41,5 +49,12 @@
...
+  quoteTokensBefore?: undefined
```
Comment (jakubswierczek) [inline features/ajna/history/types.ts:53]: Can we add here `chainId`? And then we won't need to hardcode `network` in the components

**Review by jakubswierczek — COMMENTED** [review]
- Add types to features/ajna/history/types.ts (line 52)
- Consider adding `chainId` to avoid hardcoding `network` in components (line 53)

File: features/ajna/history/types.ts (hunk 2 above)
Comment (paszkowskiDamian) [reply features/ajna/history/types.ts:53]: To the event itself?

**Review by jakubswierczek — APPROVED** [review]

**Pull request merged**